src/Services/ReorderElements.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace DNADesign\Elemental\Services;
+
+use InvalidArgumentException;
+use SilverStripe\Core\Convert;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\Queries\SQLUpdate;
+use DNADesign\Elemental\Models\BaseElement;
+
+class ReorderElements
+{
+    /**
+     * @var BaseElement $block
```

Comment:
`$block` is wrong, but you don't actually need it here - that's the description of a `@var` tag